isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml

```diff
@@ -53,8 +54,8 @@
        cameras (ONC-W1,ONC-W2). 
     </p>
     <p>
-       The wide-angle cameras each have a square 65.24 x 65.24 deg. field of view (FOV). The wide-angle 
-       camera ONC-W1 and ONC-W2 have focal lengths of
+       The wide-angle cameras each have a square 65.24 x 65.24 deg. field of view (FOV).
+       The wide-angle camera ONC-W1 and ONC-W2 have focal lengths of
```

Comment:
Suggest removing details about the instrument: Including values like the focal length here makes it easy to have incorrect info here. The instrument addendum kernel the definitive source for these values. Most people who update a kernel file will not look in the app docs to make sure they agree.

Reply:
I'm going to remove the section about the specification for the ONC-T filters as well, then.